api/notd/token_manager.py

```diff
@@ -191,3 +196,28 @@ async def update_collection_tokens(self, address: str, shouldForce: bool = False
 
     async def update_collection_tokens_deferred(self, address: str, shouldForce: bool = False):
         await self.tokenQueue.send_message(message=UpdateCollectionTokensMessageContent(address=address, shouldForce=shouldForce).to_message())
+
+    async def update_token_ownership_deferred(self, registryAddress: str, tokenId: str, shouldForce: bool = False) -> None:
```

Comment:
updates should have the object id and individual values, this structure isnt right (see the other update funcs)